math/algebraic.mbt

```diff
@@ -0,0 +1,96 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/// Compares and returns the maximum of two values.
+///
+/// Returns the second argument if the comparison determines them to be equal.
+///
+/// # Examples
+///
+/// ```
+/// print(@math.maximum(1, 2))  // output: 2
+/// print(@math.maximum(2, 2))  // output: 2
+/// ```
```

Comment:
About "code of conduct" for doc strings:

1. I used "Compares" instead of "Compare", while both forms can be found in the code base. I believe it implies "(this function) compares and returns ...".
2. I used "Examples" instead of "Example" because I do have multiple examples. I think we should stick with "Examples" (like Rust) across the repo even if it's just a single example.
3. I used `@math.maximum` instead of just `maximum` so that users could directly copy it from the documentation and run without errors.
4. I'm not sure we like the `print(...)  // output: ...` style in doc strings, it's in compatible with potential future doc tests. Though, `@assertion.assert_eq(@math.maximum(1, 2), 2)` is just too long to read, so I ended up following what the rest of this repo has been doing.

Reply:
Thanks for the suggestions, about the last one, we will consider generate example based docs directly from `test "example"`